cassandra-operator/kubernetes-resources/cassandra-operator-crd.yml

```
@@ -457,26 +486,37 @@ spec:
                       type: object
                   type: object
                 sidecarImage:
+                  description: The Docker image for the sidecar container running
+                    on each Cassandra node to expose node status.
                   type: string
               required:
               - resources
               type: object
             racks:
               items:
-                description: Rack defines the properties of a rack in the cluster
+                description: Rack defines the rack topology in the cluster, where
+                  a rack has a number of replicas located in the same physical grouping
```

```suggestion
                  a rack has a number of replicas located in the same failure domain
```

I think this fits better with Kubernetes terminology, although you could argue it's less user-friendly. What do you think?

---

Sounds good - I picked it up from the WIKI, so I'll update it there too